dace/config_schema.yml

```diff
@@ -770,6 +770,184 @@ required:
+                        description: >
+                            If matlab solver is used and problems that are not in the solver's database 
+                            are encountered, it updates the database with the new solutions.
+
+
+            # output generation
+            output:
+                type: dict
+                title: Output
+                descripion: Output format parameters
+                required:
+                    latex:
+                        type: bool
+                        title: Output in a latex format
+                        default: false
+                        description: >
+                            The I/O analysis output can be formatted as a latex table
+
+                    only_leading_term:
+                        type: bool
+                        title: Output only asymptotically leading monomial
+                        default: false
+                        description: >
+                            Output only asymptotically leading monomial
+
+                    all_params_equal:
+                        type: bool
+                        title: Simplified output result
+                        default: false
+                        description: >
+                            To simplify the output, all params can be assumed equal (e.g., N^3 instead of N*M*K)
+
+
+            # tests 
+            tests:
+                type: dict
+                title: Tests
+                decription: SOAP tests parameters
+                required:
+                    abs_test_path:
+                        type : str
+                        title : Path to the directory with "sample_sdfgs" folder
+                        default: none
+
+                    suite_name:
+                        type: str
+                        title : Name of the analyzed test suite
+                        default: ""
+                    
+                    only_selected_tests:
+                        type: list
+                        title: For a test suite, process only selected kernels
+                        default: none
+                        description: >
+                            When a specified test suite contains multiple SDFGs to analyze,
+                            (e.g., the entire Polybench), this list specifies the subset of kernels
+                            to be analyzed. If empty, all SDFGs are processed.
+                    
+                    excluded_tests:
+                        type: list
+                        title: For a test suite, omit selected kernels
+                        default: none
+                        description: >
+                            When a specified test suite contains multiple SDFGs to analyze,
+                            (e.g., the entire Polybench), this list specifies the subset of kernels
+                            to skip. If empty, all SDFGs are processed.
+
+                    einsum_string:
```

Comment:
What is the point of this being a configuration parameter? Shouldn't there be a set of fixed and representative tests for the CI?